pkg/resources/view_acceptance_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"regexp"
 	"testing"
 
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
```

Comment:
do we have a test for external column changes?
and enough tests for manipulation of columns?

Reply:
I'll add an acceptance test with internal and external changes to columns to ensure we know they behave correctly.